workspace/scm.py

```diff
@@ -193,6 +193,15 @@ def merge_branch(branch, squash=False, strategy=None):
         cmd.append('--squash')
     if strategy:
         cmd.append('--strategy=' + strategy)
+
```

Comment:
since we only need this custom message when providing a strategy lets only do this extra work if a strategy is provided.